Formula/emacs-plus@31.rb

```diff
@@ -242,17 +241,6 @@ def install
       system "gmake"
       system "gmake", "install"
     end
-
-    # Follow MacPorts and don't install ctags from Emacs. This allows Vim
```

Comment:
tbh, I don't remember why we added it
but a fair question - why did you remove it? :)

Reply:
Ctags support has been removed, so I don't think this is needed anymore.